kyuubi-server/src/main/scala/org/apache/kyuubi/server/http/authentication/AuthenticationHandler.scala

```diff
@@ -97,7 +97,11 @@ trait AuthenticationHandler {
       throw new AuthenticationException("Authorization header received from the client is empty.")
     }
 
-    val authorization = authHeader.substring(authScheme.toString.length).trim
+    var authorization = authHeader.substring(authScheme.toString.length).trim
+    // For thrift http spnego authorization, its format might be 'NEGOTIATE : $token'
```

Comment:
It's an implementation issue from Hive, I suggest you report this to the Hive community and link the JIRA Ticket here.